cost-model/src/cost_model.rs

```diff
@@ -105,8 +115,49 @@ impl CostModel {
         }
     }
 
+    /// Return an estimated total cost for a transaction given its':
+    /// - `meta` - transaction meta
+    /// - `instructions` - transaction instructions
+    /// - `num_write_locks` - number of requested write locks
+    ///
+    /// This assumes ALL features are activated.
+    /// This is intended to be used as an estimate of cost ONLY for the purposes of
+    /// prioritization and packing.
+    pub fn estimate_cost<'a>(
+        meta: &impl StaticMeta,
+        instructions: impl Iterator<Item = (&'a Pubkey, SVMInstruction<'a>)> + Clone,
+        num_write_locks: u64,
+    ) -> u64 {
+        if meta.is_simple_vote_transaction() {
+            return TransactionCost::SimpleVote { transaction: meta }.sum();
+        }
+
+        lazy_static! {
+            static ref ALL_FEATURES: FeatureSet = FeatureSet::all_enabled();
+        };
```

Comment:
As suggested above, can let call side decided what feature_set to use.

Reply:
[a67c609](https://github.com/anza-xyz/agave/pull/3986/commits/a67c609a4ef8eff7716e11975b79980493de2e1c)